afs/afs-ext-base/src/main/java/com/powsybl/afs/ext/base/events/CaseImported.java

```diff
@@ -0,0 +1,52 @@
+/**
+ * Copyright (c) 2018, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.afs.ext.base.events;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.powsybl.afs.storage.events.NodeEvent;
+import com.powsybl.afs.storage.events.NodeEventType;
+
+import java.util.Objects;
+
+/**
+ * @author Chamseddine Benhamed <chamseddine.benhamed at rte-france.com>
+ */
+public class CaseImported extends NodeEvent {
+
+    @JsonProperty("parentId")
+    protected final String parentId;
```

Comment:
It is a business oriented event, instead of parentId it would be better to send to full path in addition to the id.